Tests/SonarQube.MSBuild.Tasks.UnitTests/AttachBuildWrapperTests.cs

```diff
@@ -17,12 +18,17 @@ namespace SonarQube.MSBuild.Tasks.UnitTests
     [TestClass]
     public class AttachBuildWrapperTests
     {
+        /// <summary>
+        /// Name of the C++ plugin property that specifies where the build wrapper output is written
+        /// </summary>
+        private const string BuildOutputSettingName = "sonar.cfamily.build-wrapper-output";
+
         public TestContext TestContext { get; set; }
 
         #region Tests
 
         [TestMethod]
-        public void AttachBW_MissingFiles_Fails()
+        public void AttachBW_MissingBinaryFiles_TaskFails()
```

Comment:
Changed the name to make it clearer that it's only checking for missing binary files. A missing config file is covered by a new test.